### PR TITLE
Add troubleshooting notes for TB

### PR DESCRIPTION
**Pull request by mudler**

### CODE REVIEW

File: content/en/docs/Reference/troubleshooting.md [hunk 1]
```diff
@@ -98,4 +98,67 @@ Some small things you can do to provide us with the best information possible:
...
+##### Checking if Secure Boot is Enabled
+
+Secure Boot is a security standard that helps to ensure that a device boots using only software that is trusted by the Original Equipment Manufacturer (OEM). When Secure Boot is enabled, the firmware checks each piece of boot software for a valid digital signature.
+
+##### Command to Check Secure Boot Status
+
+To check if Secure Boot is enabled on your system, use the following command:
+
+```bash
+dmesg | grep -i secure
```
Comment (Itxaka) [inline content/en/docs/Reference/troubleshooting.md:161]: We got the same info on kairos-agent state now!

**Review by Itxaka — APPROVED** [review]

File: content/en/docs/Reference/troubleshooting.md (hunk 1 above)
Comment (jimmykarily) [reply content/en/docs/Reference/troubleshooting.md:161]: also `sbctl status`

**Review by jimmykarily — APPROVED** [review]
nice!

File: content/en/docs/Reference/troubleshooting.md (hunk 1 above)
Comment (mudler) [reply content/en/docs/Reference/troubleshooting.md:161]: nice, let's keep this page up-to-date with the kairos-agent commands too :+1:

**Pull request merged**